### PR TITLE
Update Spanish translation

### PR DESCRIPTION
**Pull request by dcapillae**
Spanish translation updated according to the latest changes in English.

### CODE REVIEW

File: openlibrary/i18n/es/messages.po [hunk 1]
```diff
@@ -1471,234 +1528,277 @@ msgstr ""
...
 msgid ""
-"Displaying stats about <strong>%d</strong> books. Note all charts show only "
-"the top 20 bars. Note reading log stats are private."
+"Displaying stats about <strong>%(works_count)d</strong> books. Note all "
+"charts show only the top 20 bars. Note reading log stats are private."
 msgstr ""
-"Muestra estadísticas sobre <strong>%d</strong> libros. Ten en cuenta que "
-"todos los gráficos muestran solo las 20 primeras barras. Ten en cuenta que "
-"las estadísticas del registro de lectura son privadas."
+"Visualización de estadísticas sobre strong>%(works_count)d</strong> libros. "
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:1558]: They should all be small stuff like this:

```suggestion
"Visualización de estadísticas sobre <strong>%(works_count)d</strong> libros. "
```

Comment (dcapillae) [reply openlibrary/i18n/es/messages.po]: I am not sure how I should proceed with the other issue (<strong> was my fault, sorry). In the message.pot file some html codes are also missing (</i>). Should we include them before in message.pot? 

I also found a strange string in message.pot that I didn't know if I should translate (line 1074). Shouldn't some code be used in "TITLE"?

> "Are you sure you want to leave the waiting list "
> "of<br/><strong>TITLE</strong>?"

**Comment by cdrini** [comment]
Ah which translations are missing `</i>` ? I can fix those!

We should create a new issue for the `TITLE` one ; for now, leave `TITLE` but translate the rest :+1:

**Comment by dcapillae** [comment]
There are three strings that are missing the closing </i> code in the message.pot file, the strings starting with `For example: <i>` (lines 2864, 2875, 2886, 2897). I did not add it to the translation because they are also missing in the .pot file and I preferred to leave it the same so as not to break anything.

These are the translations that have not passed HTML validation, the translation strings starting with `Por ejemplo: <i>` (lines 3059, 3073, 3087, 3101) in the Spanish message.po file.

File: openlibrary/i18n/es/messages.po [hunk 2]
```diff
@@ -2407,50 +2706,205 @@ msgstr ""
...
 msgid "First published in %(year)d"
 msgstr "Publicado por primera vez en %(year)d"
 
+#: books/check.html
+msgid "None of these match the book I want to add."
+msgstr "Ninguno de estos coincide con el libro que quiero añadir."
+
+#: books/check.html
+msgid "Continue</a>."
+msgstr "Continuar</a>."
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:2732]:
```suggestion
msgid "Continue"
msgstr "Continuar"
```

File: openlibrary/i18n/es/messages.po [hunk 3]
```diff
@@ -2407,50 +2706,205 @@ msgstr ""
...
+"por la Biblioteca del Congreso."
+
+#: books/daisy.html
+#, python-format
+msgid ""
+"<strong>Download the DAISY.zip</strong></a> for <a "
+"href=\"%(url)s\">%(title)s</a>"
+msgstr ""
+"<strong>Decargar el DAISY.zip</strong></a>para <a "
+"href=\"%(url)s\">%(title)s</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:2775]:
```suggestion
msgid ""
"<a href=\"%(daisy_url)s\"><strong>Download the DAISY.zip</strong></a> for"
" <a href=\"%(url)s\">%(title)s</a>"
msgstr ""
"<a href=\"%(daisy_url)s\"><strong>Decargar el DAISY.zip</strong></a>para <a "
"href=\"%(url)s\">%(title)s</a>"
```

File: openlibrary/i18n/es/messages.po [hunk 4]
```diff
@@ -2499,282 +2953,317 @@ msgstr ""
...
-msgstr "Por ejemplo:"
+#: books/edit/about.html
+msgid ""
+"For example: <i><a href=\"/subjects/cheese\">cheese</a>, <a href=\"/subjects/"
+"roman_empire\">Roman Empire</a>, <a href=\"/subjects/"
+"psychology\">psychology</a>"
+msgstr ""
+"Por ejemplo: <i><a href=\"/subjects/cheese\">queso</a>, <a href=\"/subjects/"
+"roman_empire\">Imperio Romano</a>, <a href=\"/subjects/"
+"psychology\">psicología</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:3061]:
```suggestion
"psychology\">psicología</a></i>"
```

File: openlibrary/i18n/es/messages.po [hunk 5]
```diff
@@ -2499,282 +2953,317 @@ msgstr ""
...
-#: books/edit/about.html:59
+#: books/edit/about.html
+msgid ""
+"For example: <i><a href=\"/subjects/person:Theodore_Roosevelt\">Theodore "
+"Roosevelt</a>, <a href=\"/subjects/person:Julian_of_Norwich\">Julian of "
+"Norwich</a>, <a href=\"/subjects/person:Tintin\">Tintin</a>"
+msgstr ""
+"Por ejemplo: <i><a href=\"/subjects/person:Theodore_Roosevelt\">Theodore "
+"Roosevelt</a>, <a href=\"/subjects/person:Julian_of_Norwich\">Juliana de "
+"Norwich</a>, <a href=\"/subjects/person:Tintin\">Tintín</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:3075]:
```suggestion
"Norwich</a>, <a href=\"/subjects/person:Tintin\">Tintín</a></i>"
```

File: openlibrary/i18n/es/messages.po [hunk 6]
```diff
@@ -2499,282 +2953,317 @@ msgstr ""
...
-#: books/edit/about.html:70
+#: books/edit/about.html
+msgid ""
+"For example: <i><a href=\"/subjects/place:London\">London</a>, <a href=\"/"
+"subjects/place:Atlantis\">Atlantis</a>, <a href=\"/subjects/place:"
+"Omaha\">Omaha</a>"
+msgstr ""
+"Por ejemplo: <i><a href=\"/subjects/place:London\">Londres</a>, <a href=\"/"
+"subjects/place:Atlantis\">Atlántida</a>, <a href=\"/subjects/place:"
+"Omaha\">Omaha</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:3089]:
```suggestion
"Omaha\">Omaha</a></i>"
```

File: openlibrary/i18n/es/messages.po [hunk 7]
```diff
@@ -2499,282 +2953,317 @@ msgstr ""
...
-#: books/edit/edition.html:22
+#: books/edit/about.html
+msgid ""
+"For example: <i><a href=\"/subjects/time:1984\">1984</a>, <a href=\"/"
+"subjects/time:the_middle_ages\">The Middle Ages</a>, <a href=\"/subjects/"
+"time:1810-1890\">1810-1890</a>"
+msgstr ""
+"Por ejemplo: <i><a href=\"/subjects/time:1984\">1984</a>, <a href=\"/"
+"subjects/time:the_middle_ages\">Edad Media</a>, <a href=\"/subjects/"
+"time:1810-1890\">1810-1890</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:3103]:
```suggestion
"time:1810-1890\">1810-1890</a></i>"
```

**Review by cdrini — COMMENTED** [review]
Ok I fixed most of these in #9214 ; there's still one erroring which I'm investigating :+1:

File: openlibrary/i18n/es/messages.po [hunk 8]
```diff
@@ -1471,234 +1528,277 @@ msgstr ""
...
 msgid "Works by Author Country of Birth"
 msgstr "Obras por país de nacimiento del autor"
 
-#: account/readinglog_stats.html:134
+#: account/readinglog_stats.html
 msgid ""
 "Demographic statistics powered by <a href=\"https://www.wikidata.org/"
 "\">Wikidata</a>. Here's a <a id=\"wd-query-sample\" href=\"\">sample</a> of "
-"the query used."
+"the query used. <br>Improve these stats by adding the Wikidata author "
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:1586]:
```suggestion
"the query used. <br />Improve these stats by adding the Wikidata author "
```

File: openlibrary/i18n/es/messages.po [hunk 9]
```diff
@@ -1471,234 +1528,277 @@ msgstr ""
...
+"identifier following <a href=\"https://openlibrary.org/help/faq/editing."
+"en#author-identifiers-purpose\"> these instructions</a>."
 msgstr ""
-"Estadísticas demográficas con la ayuda de <a href=\"https://www.wikidata.org/"
+"Estadísticas demográficas basadas en <a href=\"https://www.wikidata.org/"
 "\">Wikidata</a>. Aquí hay un <a id=\"wd-query-sample\" href=\"\">ejemplo</a> "
-"de la consulta empleada."
+"de la consulta empleada. <br>Mejora estas estadísticas añadiendo el "
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:1592]:
```suggestion
"de la consulta empleada. <br />Mejora estas estadísticas añadiendo el "
```

File: openlibrary/i18n/es/messages.po [hunk 10]
```diff
@@ -2499,282 +2953,317 @@ msgstr ""
...
-#: books/edit/edition.html:355 books/edit/edition.html:369
-#: books/edit/edition.html:577 books/edit/excerpts.html:24
-#: books/edit/web.html:29 type/author/edit.html:113
-msgid "For example:"
-msgstr "Por ejemplo:"
+#: books/edit/about.html
+msgid ""
+"For example: <i><a href=\"/subjects/cheese\">cheese</a>, <a href=\"/subjects/"
+"roman_empire\">Roman Empire</a>, <a href=\"/subjects/"
+"psychology\">psychology</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:3057]:
```suggestion
"psychology\">psychology</a></i>"
```

File: openlibrary/i18n/es/messages.po [hunk 11]
```diff
@@ -2499,282 +2953,317 @@ msgstr ""
...
+#: books/edit/about.html
 msgid "A person? Or, people?"
 msgstr "¿Una persona? ¿O personas?"
 
-#: books/edit/about.html:59
+#: books/edit/about.html
+msgid ""
+"For example: <i><a href=\"/subjects/person:Theodore_Roosevelt\">Theodore "
+"Roosevelt</a>, <a href=\"/subjects/person:Julian_of_Norwich\">Julian of "
+"Norwich</a>, <a href=\"/subjects/person:Tintin\">Tintin</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:3071]:
```suggestion
"Norwich</a>, <a href=\"/subjects/person:Tintin\">Tintin</a></i>"
```

File: openlibrary/i18n/es/messages.po [hunk 12]
```diff
@@ -2499,282 +2953,317 @@ msgstr ""
...
+#: books/edit/about.html
 msgid "Note any places mentioned?"
 msgstr "Anota cualquier lugar mencionado"
 
-#: books/edit/about.html:70
+#: books/edit/about.html
+msgid ""
+"For example: <i><a href=\"/subjects/place:London\">London</a>, <a href=\"/"
+"subjects/place:Atlantis\">Atlantis</a>, <a href=\"/subjects/place:"
+"Omaha\">Omaha</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:3085]:
```suggestion
"Omaha\">Omaha</a></i>"
```

File: openlibrary/i18n/es/messages.po [hunk 13]
```diff
@@ -2499,282 +2953,317 @@ msgstr ""
...
+#: books/edit/about.html
 msgid "When is it set or about?"
 msgstr "¿Cuándo se desarrolla o de qué época se ocupa?"
 
-#: books/edit/edition.html:22
+#: books/edit/about.html
+msgid ""
+"For example: <i><a href=\"/subjects/time:1984\">1984</a>, <a href=\"/"
+"subjects/time:the_middle_ages\">The Middle Ages</a>, <a href=\"/subjects/"
+"time:1810-1890\">1810-1890</a>"
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:3099]:
```suggestion
"time:1810-1890\">1810-1890</a></i>"
```

**Review by cdrini — APPROVED** [review]
Ok this should be the last of them!

**Pull request merged**